contracts/SnappAuction.sol

```diff
@@ -179,20 +182,20 @@ contract SnappAuction is SnappBase {
         auctions[auctionIndex].size++;
     }
 
-    function placeMultiSellOrder(
-        uint8[] memory buyTokens,
-        uint8[] memory sellTokens,
-        uint128[] memory buyAmounts,
-        uint128[] memory sellAmounts
-    ) public {
-        uint numOrders = buyTokens.length;
-        require(numOrders == sellTokens.length, "numOrders != sellTokens length");
-        require(numOrders == buyAmounts.length, "numOrders != buyAmounts length");
-        require(numOrders == sellAmounts.length, "numOrders != sellAmounts length");
+    function placeMultiSellOrder(bytes memory packedOrders) public {
+        // Note that this could result failure of all orders if even one fails.
+        require(packedOrders.length % 27 == 0, "Each order should be packed in 27 bytes!");
```

Comment:
Why 27? Isn't it 2 for accountId, 1 for buy + 1 for sell token & 12 for buy + 12 for sell token? That would be 28 bytes, wouldn't it?

Reply:
Yes it should be 28... Not sure where I lost one.